test/Microsoft.AspNetCore.Mvc.Razor.Host.Test/DictionaryPrefixTestTagHelper.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.AspNetCore.Mvc.ViewFeatures;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+using System.Collections.Generic;
+
+namespace Microsoft.AspNetCore.Mvc.Razor.Host.Test
+{
+    [HtmlTargetElement(Attributes = "prefix-*")]
+    public class DictionaryPrefixTestTagHelper : TagHelper
```

Comment:
The generated code below shows this tag helper is not bound. Please figure out why and fix it.